prow/cluster/cert-manager-deployment.yaml

```diff
@@ -0,0 +1,259 @@
+---
+# Source: cert-manager/templates/00-namespace.yaml
+
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: "cert-manager"
+  labels:
+    name: "cert-manager"
+    certmanager.k8s.io/disable-validation: "true"
+
+---
+# Source: cert-manager/templates/serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cert-manager
+  namespace: "cert-manager"
+  labels:
+    app: cert-manager
+    chart: cert-manager-v0.6.0-dev.2
+    release: cert-manager
+    heritage: Tiller
+---
+# Source: cert-manager/templates/certificate-crd.yaml
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: certificates.certmanager.k8s.io
+  annotations:
+    "helm.sh/hook": crd-install
+  labels:
+    app: cert-manager
+    chart: cert-manager-v0.6.0-dev.2
```

Comment:
Looks like you've pulled these manifests from the master branch - you should be using `release-0.5` instead, unless you're feeling super :this_is_fine: 😄